randovania/games/dread/json_data/Burenia.txt

```diff
@@ -295,7 +298,9 @@ Extra - asset_id: collision_camera_002
   > Door to Underwater Horseshoe
       After Burenia - db_reg_aq_001 or After Burenia - db_reg_aq_002
   > Fan Platform - Below
-      Gravity Suit or Water Bomb Jump (Beginner)
+      Any of the following:
+          Gravity Suit or Water Bomb Jump (Beginner)
+          After Burenia - db_reg_aq_001 and After Burenia - db_reg_aq_002
```

Comment:
Remove these blob requirements. The "Raft - Top Water Level" is a node for the same altitude as the raft's highest point - i.e. neither blob is destroyed

The logic for destroying the blobs to get to the higher level are already accounted for with "Door to Underwater Horseshoe -> Raft - Top Water Level" and vice versa.

Reply:
EDIT: never mind, I see why you added these.